src/albums.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+)
+
+// TODO : next feature
+// *  Add possibility to reload a directory or add new directory at the runtime
+
+// Albums Type  contain photos
+type Albums struct {
+	lstPhotos      []*Photo
+	directoryPaths []string
+}
+
+// NewAlbum , return Albums struct
+func NewAlbum(directoryPaths []string) *Albums {
+	return &Albums{
+		directoryPaths: directoryPaths,
+	}
+}
+
+// LoadPhotosInAlbums , iterate for each directory and load photos
+func (a *Albums) LoadPhotosInAlbums() (int, error) {
+	// Loop in each directory and load picture files
+	// TODO: need improve regex
+	libRegEx, err := regexp.Compile("^.+\\.(jpg)$")
+	if err != nil {
+		return 0, err
+	}
+
+	// stop if no directory was provided
+	if len(a.directoryPaths) < 1 {
+		return 0, fmt.Errorf("No directory path provided")
+	}
+
+	// loop to extract picture for all directory
+	for _, v := range a.directoryPaths {
+		err = filepath.Walk(v, func(path string, info os.FileInfo, errFile error) error {
+			if errFile == nil && libRegEx.MatchString(info.Name()) {
+				err = a.chargePicInAlbum(path)
+				if err != nil {
+					fmt.Printf("Error with file %s error extracting tags : %v", path, err)
+				}
```

Comment:
again multiple error management will be better here, I do not want to block the process but have a better information report on what went wrong.